src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -79,6 +86,31 @@
       (add-acl m)
       (add-query-params m)))
 
+(defn in-valid-methods?
+  "Is the given method contained in the list of valid methods?"
+  [m]
+  (let [method (str/lower-case m)]
+    (boolean (some #(= method %) valid-methods))))
+
+(defn valid-method?
+  "Returns true if the given rule contains either a valid method, or no speicfied
+  method."
+  [rule]
+  (if (nil? (:match-request rule)) false
```

Comment:
Yeah, true.  Looks like we do already have an explicit check for the existence of `:match-request` in `validate-auth-config-rule!` so should be good there already.